MAINTAINERS.md

```diff
@@ -77,8 +85,54 @@ Make frequent project releases to the community.
 
 ### Promote Other Maintainers
 
-Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers.
+Assist, add, and remove [MAINTAINERS](MAINTAINERS.md). Exercise good judgement, and propose high quality contributors to become co-maintainers. See [Becoming a Maintainer](#becoming-a-maintainer) for more information.
 
 ### Describe the Repo
 
 Make sure the repo has a well-written, accurate, and complete description. See [opensearch-project/.github#38](https://github.com/opensearch-project/.github/issues/38) for some helpful tips to describe your repo.
+
+## Becoming a Maintainer
+
+You can become a maintainer by actively [contributing](CONTRIBUTING.md) to any project, and being nominated by an existing maintainer.
+
+### Nomination
+
+Any current maintainer starts a private e-mail thread (until we have a better mechanism, e-mail addresses can usually be found via MAINTAINERS.md + DCO) with all other maintainers on that repository to discuss nomination using the template below. In order to be approved, at least three positive maintainer votes are necessary, and no vetoes. In rare cases when there are fewer than three maintainers, the positive votes from all maintainers are required. Any disagreements can be escalated to the repo admin.
+
+The nomination should clearly identify the person with their real name and a link to their GitHub profile, and the rationale for the nomination, with concrete example contributions.
+
+### Interest
+
+Upon receiving at least three positive maintainer votes, and no vetoes, from other maintainers within a maximum period of two weeks, the nominating maintainer asks a potential nominee whether they might be interested in becoming a maintainer on the repository, which happens via private e-mail message, but may also be a comment to a pull request, following a substantial contribution.
```

Comment:
s/asks a potential nominee/asks the potential nominee

`within a maximum period of two weeks` is odd -- did you mean minimum period maybe?   Or maybe "at least one week and at most two weeks"?

If it is meant to be minimum, two weeks is too long imo.

I still think we should remove the `, but may also be ...` clause -- asking the nominee should be a private email only -- people decline for all sorts of important personal reasons, and that part should remain private as well.

Reply:
It's meant to say that maintainers will wait for max 2 weeks to collect votes and vetoes. Hopefully maintainers are active enough to respond quicker than that. So it's not a minimum, nor is it at least one week and at most two. I don't know how to wordsmith this better.

Reply:
I don't care enough to allow asking in public, removed. 

https://github.com/opensearch-project/.github/pull/59/commits/732388472c9f100465bfa3fac33e73a8e35f99ec